src/platform/site-wide/wc-loader.js

```diff
@@ -0,0 +1,6 @@
+import 'web-components/dist/component-library/component-library.css';
```

Comment:
This contains some [global CSS variables](https://github.com/department-of-veterans-affairs/component-library/blob/cf7a0b052041d423e934873e8be506c2d755f892/src/global/variables.css#L5-L12) for our components to use.